src/client/routes/nfc/NfcTable.test.tsx

```diff
@@ -35,6 +35,7 @@ test.skip('Test NfcTable', () => {
 									name: 'test-event',
 									eventType: 'TEST',
 									startTimestamp: 0,
+									duration: 0,
```

Comment:
why does a test event not have any duration?

Reply:
I think all these fields are just placeholders